openedx/core/djangoapps/content/learning_sequences/tasks.py

```diff
@@ -22,6 +23,14 @@ def update_from_modulestore(course_key):
     We should move this out so that learning_sequences does not depend on
     ModuleStore.
     """
+    course_outline_data = get_outline_from_modulestore(course_key)
+    replace_course_outline(course_outline_data)
+
+
+def get_outline_from_modulestore(course_key):
```

Comment:
I like the way this is separated out from the `update_from_modulestore` method the way you have done it. Curious as to why you did so though.

Reply:
@pcockwell David suggested it himself https://github.com/edx/edx-platform/pull/24417#discussion_r451278984

Reply:
Thanks!